angelsbioprocessing/locale/en/bio-processing.cfg

```diff
@@ -491,41 +491,41 @@ bio-refugium-biter=Biter refugium
 bio-refugium-hogger=Hogger refugium
 
 [item-description]
-composter="Used to turn any organic material into compost."
-solid-saw="He saw, he cut, and he shattered."
```

Comment:
This one in particular should still be a quote, since it's a reference and not a description

Reply:
Should I change it for this pull request?

Reply:
Yes please, that would be the easiest thing, I can also do this right before merging it. Just don't have a good PC atm to test this, hence why I didn't merge it yet

Reply:
done.